Makefile

```diff
@@ -73,7 +73,12 @@ DEV_SUFFIX=-dev
 DEPLOYS_API_KEY?=gateway4deployes
 
 # this should use the same version as our prod RDS database
-LOGGER_POSTGRES_TAG?=15
+CONTROL_DB_POSTGRES_TAG?=15
+CONTROL_DB_POSTGRES_PASSWORD?=postgres
+CONTROL_DB_POSTGRES_URI?=postgres://postgres:${CONTROL_DB_POSTGRES_PASSWORD}@control-postgres:5432/postgres
+
+# use the same version as control DB
+LOGGER_POSTGRES_TAG?=${CONTROL_DB_POSTGRES_TAG}
```

Comment:
While it's highly likely, this shouldn't necessarily be the same version as the control DB.
```suggestion
# this should use the same version as our prod RDS logger database
LOGGER_POSTGRES_TAG?=15
```